gateway-js/src/executeQueryPlan.ts

```diff
@@ -106,7 +107,22 @@ export async function executeQueryPlan<TContext>(
           });
           data = executionResult.data;
           if (executionResult.errors?.length) {
-            errors.push(...executionResult.errors)
+            if (removedTypes && removedTypes.size > 0) {
+              const anyInaccessibleTypename = new RegExp(
+                [...removedTypes]
+                  .map((namedType) => '("' + namedType.name + '")')
+                  .join('|'),
+                'g',
+              );
+              executionResult.errors.forEach(
+                (e) =>
+                  (e.message = e.message.replace(
```

Comment:
Currently, I believe that `graphql-js` can only return this type name a single time in an error message. 
 Should we be defensive for the case that might change by using `String.prototype.replaceAll` here?

Reply:
it's already replacing-all, the `'g'` when we construct the regexp makes it a global pattern, calling `.replace` with a global pattern replaces all instances of it

Reply:
Ah, thanks/disregard my suggestion. I totally missed the `g` modifier on the `RegExp` call; I suppose I'm better at reading the `//modifier`-framed syntax.

Reply:
Tangentially related nuanced note I just recalled: `replaceAll` throws if you pass it a `RegExp` without the `g` modifier, whereas `replace` accepts either with or without the global modifier.  

But the string values behave according to the function name (_all_ or _not all_).